.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: 'close stale issues/PRs'
+on:
+  schedule:
+    - cron: '* */6 * * *'
+  workflow_dispatch:
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@87c2b794b9b47a9bec68ae03c01aeb572ffebdb1
+        with:
+          repo-token: ${{ github.token }}
+          days-before-stale: 21
+          days-before-close: 7
+          only-labels: ""
+          operations-per-run: 100
+          remove-stale-when-updated: false
+          debug-only: false
+          ascending: false
+
+          exempt-issue-labels: "Triage: Accepted"
+          stale-issue-label: "Triage: Stale"
+          stale-issue-message: "This message is required, but you shouldn't see it."
+          skip-stale-issue-message: true
+          close-issue-label: ""
+          close-issue-message: ""
+
+          exempt-pr-labels: "Triage: Accepted"
```

Comment:
I think for PRs we may also use WIP?

Reply:
I see `wip` in [`sentry-docs`](https://github.com/getsentry/sentry-docs/labels) but not in [`sentry`](https://github.com/getsentry/sentry/labels) or [`onpremise`](https://github.com/getsentry/onpremise/labels). On the one hand, `wip` labels are obsolete since GitHub implemented [Draft PRs](https://github.blog/2019-02-14-introducing-draft-pull-requests/). On the other, I would think that draft/WIP PRs are precisely what we _want_ to prune if they go stale.

Reply:
On the third hand(?), `sentry` has a `Status: Accepted` label, so I'm going to switch to that.